src/Lykke.Service.PaySettlement/Cqrs/ConfirmationSavedEvent.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ProtoBuf;
+
+namespace Lykke.Service.PaySettlement.Cqrs
+{
+    [ProtoContract]
+    public class ConfirmationSavedEvent
```

Comment:
It's better to use the TxDetector contracts; not to copy-paste contracts code.

Reply:
Is there any shared contracts from TxDetector?